api/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -50,6 +50,31 @@ type NginxIngressControllerSpec struct {
 	// will be from the Azure LoadBalancer annotations here https://cloud-provider-azure.sigs.k8s.io/topics/loadbalancer/#loadbalancer-annotations
 	// +optional
 	LoadBalancerAnnotations map[string]string `json:"loadBalancerAnnotations,omitempty"`
+
+	// DefaultSSLCertificate is a struct with a secret with the fields namespace and name which is used to configure the NginxIngressController being managed via this CRD.
```

Comment:
DefaultSSLCertificate defines whether the NginxIngressController should use a certain SSL certificate by default. If this field is not defined, no default certificate will be used.

Reword to ^